### ISSUE TITLE
Change Alternative Name field for experiments to something else.

### ISSUE THREAD

**Issue by charliepascoe** [ES-DOC/esdoc-docs#271]
Suggestions:
- alias
- AKA
- working name
- previous name

The ES-DOC experiment documentation lists all the "other_names" that an experiment has been known as.  In the online ES-DOC experiment viewer these other_names are listed as: 
"Alternative Name" on the list of experiments https://documentation.es-doc.org/cmip6/experiments 
and as: 
"Alternative Names" on an individual experiment document e.g. https://documentation.es-doc.org/cmip6/experiments/pa-pdSIC.

Karl Taylor doesn't like the use of "Alternative Name(s)" to describe the other_names that experiments are/have been known by, he doesn't want modelling groups to use these other_names when describing their data.  So we need to come up with a name that makes it clear that the other_names for the experiments are only for reference.  

Karl has suggested that we use "previous name" but I don't like this choice because some of the other_names for experiments are still in use by the MIP teams within the scope of their MIP. For example the PAMIP experiments are referred to using a numbering system within the PAMIP GMD paper https://www.geosci-model-dev-discuss.net/gmd-2018-82/gmd-2018-82.pdf 
 
My preference is to use "working name"

**Comment by martinjuckes** [comment]
There appear to be two use-cases covered by this field: if I've understood correctly, it contains information about **obsolete** terms which may have been used in the literature, but also, according to   Charlotte's comment above, some genuine aliases which are still in use within MIPs (who may have developed their own portals etc for managing their workflows). If it is too late to split these by adding an additional field for the obsolete terms, perhaps we could call the field `Alias` and distinguish the obsolete terms by adding an asterisk (e.g. `volc-pinatubo-surf` would have `alias` `*volcEq-surf, *VolsShort20EQsurf`).

Alternatively, perhaps give it a longer name `Aliases and Obsolete Names`.

**Comment by charliepascoe** [comment]
I agree, splitting the other_name field into alias and obsolete_names would be the best option.

That would require an update to the CIM, and also teasing out aliases from other names.  
I've helped us a bit, in that the first item in the list of other_names is either the alias or the most recently used obsolete name.

We then get into the question of what is an alias?  
I would say an alias is any name that an experiment is also known by in the articles that the MIPs submitted to GMD as the definitive text to describe their MIPs for CMIP6.

Note that aliases will not necessarily be unique, for instance four of the CDRMIP CMIP6 experiments are considered by CDRMIP to be parts of the single experiment: CDR-yr2010-pulse experiment.

**Comment by taylor13** [comment]
My main concern is that users might think that they can use the "alias" name in using CMIP6 infrastructure.  The infrastructure *only* understands (except, perhaps, ES-DOCs) the official experiment and activity i.d.'s as recorded in the .json files.  Is there any way to make clear that "alias" cannot be used as a substitute for experiment_id?

**Comment by charliepascoe** [comment]
I have separated out the other_names field into an alias field and a previous_names field.
I then reviewed the GMD articles to decide which names should be listed as aliases.
Here are examples of reasons why names have been listed as an alias:

Simply a matter of a difference in case	
e.g. PMIP experiment midPliocene-eoi400 which is referred to as midPliocene-Eoi400 in their GMD paper

A name has been updated in the CMIP6 CV so that it follows the naming convention for experiments with the same type of forcing but the GMD paper still uses the old convention
e.g. HighResMIP experiment highresSST-4xCO2 which is referred to as highresSST-4co2 in their GMD paper

This name appeared to have name length restriction imposed by the MIP but not by the CMIP6 CV
e.g. LUMIP deforest-glob which is referred to as deforest-globe in their GMD.

As far as I can tell GeoMIP hasn’t included CMIP6 names in their GMD paper
e.g. futureSST-4xCO2-solar is referred to as G1extSlice2

Some MIPs have their own name for an experiment as well as a CMIP6 name.  Whilst they refer to the CMIP6 name within their paper other MIPs refer to the MIP specific name instead.
e.g.
DAMIP refers to MIP specific RFMIP names
RFMIP refers to MIP specific DCPP names
e.g.
RFMIP refers to DCPP C3.4 instead of dcppC-forecast-addPinatubo

PAMIP lists the CMIP6 names in the tables of their GMD paper but within the text they refer to their experiments by their own MIP specific names.
e.g. 1.1 instead of pdSST-pdSIC

CDRMIP lists the CMIP6 names as well as their own but they have a wider usage for the term “experiment".  For instance they refer to the experiment CDR-ocean-alk which in CMIP6 land is two experiments: esm-ssp585-ocn-alk and esm-ssp585-ocn-alk-stop

LS3MIP had an experiment separated into two by the CMIP6 CV team, they still refer to the experiment Land-Future which has become land-ssp585 and land-ssp434 in CMIP6 land.
